Week2/homework/maartjes_work.js

```diff
@@ -44,4 +44,22 @@ const tuesday = [
 
 const tasks = monday.concat(tuesday);
 
-// Add your code here
+//step 1. duration in hour
+const hours = tasks.map(hour => {
+  return hour.duration/60;
+});
+
+//step 2. durations less than 2 hours
+const moreThan2Hours = hours.filter(x => {
+  return x >= 2;
+});
+
+//step 3 total salary
+const salary = moreThan2Hours.map(earn => earn * 17)
+.reduce((accumulator, currentValue) => accumulator + currentValue);
+
+//step 4 rounded salary
+console.log(`€${salary.toFixed(0)}`);
```

Comment:
I would be inclined to make the parameter to `.toFixed` 2, so you have more of a currency feel with two decimal places.